app/utility/transforms/CollapsedMetrics.js

```diff
@@ -364,6 +365,41 @@ export const getSetLossOfDurations = (set) => {
     return getSetLossMetrics(durations);
 };
 
+export const getRPE1rm = (set) => {
+    let reps = [];
+
+    set.reps.forEach((rep) => {
+        if (rep.isValid && !rep.removed) {
+            reps.push(rep);
+        }
+    });
+
+    const weight = set.weight;
+    const rpe = Number(set.rpe.replace(',','.'));
+
+    if (rpe < 6.5 || !rpe || reps.length === 0 || !rpe) {
```

Comment:
- [ ] you check for !rpe twice, only need to check once
- [ ] missing weight check